kaitai_utils.py

```diff
@@ -1,40 +1,37 @@
 import enum
```

Comment:
Does this keep working for .ksy like Zip where the output of the compiler are multiple .py files?

Reply:
Yes, it does! (I checked that as well)